external/avro/src/main/scala/org/apache/spark/sql/avro/AvroSerializer.scala

```diff
@@ -93,7 +99,10 @@ private[sql] class AvroSerializer(
 
   private lazy val decimalConversions = new DecimalConversion()
 
-  private def newConverter(catalystType: DataType, avroType: Schema): Converter = {
+  private def newConverter(catalystType: DataType, avroType: Schema,
+    catalystPath: Seq[String], avroPath: Seq[String]): Converter = {
```

Comment:
shall we use a consistent name? either `sqlPath` or `catalystPath` is fine.

Reply:
I used `sqlPath` in `AvroDeserializer` because `getRecordWriter` was already calling the Catalyst datatype `sqlType` and I wanted to be consistent. But now I see that even `AvroDeserializer#newWriter` is referring to this as `catalystType`, so I will make it consistent throughout. Thanks for the callout.